hawkbit-rest/hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiDeploymentBaseTest.java

```diff
@@ -700,20 +694,23 @@ public void missingResultAttributeInFeedbackReturnsBadRequest() throws Exception
         assignDistributionSet(ds.getId(), "1080");
         final Action action = deploymentManagement.findActionsByTarget(target.getControllerId(), PAGE).getContent()
                 .get(0);
-        final String missingResultInFeedback = JsonBuilder.missingResultInFeedback(action.getId().toString(), "closed",
-                "test");
-        postDeploymentFeedback("1080", action.getId(), missingResultInFeedback,
-                status().isBadRequest()).andExpect(jsonPath("$.*", hasSize(3))).andExpect(
-                        jsonPath("$.exceptionClass", equalTo(MessageNotReadableException.class.getCanonicalName())));
+
+        final DdiStatus ddiStatus = new DdiStatus(DdiStatus.ExecutionStatus.CLOSED, null,
+                Collections.singletonList("test"));
+        final String missingResultInFeedback = objectMapper
+                .writeValueAsString(new DdiActionFeedback("20220815T121314", ddiStatus));
```

Comment:
can't we somehow reuse the `getJsonActionFeedback` here?

Reply:
Now this method also uses `getJsonActionFeedback`